### DISCUSSION TITLE
Can't access GitHub Copilot with valid student account.

### DISCUSSION THREAD

**Discussion by ameu00** [community/community#145208] [GitHub Education]
### Hi everyone,

Hi,
I've submitted a request to renew my status as a student a week ago (2024-11-13). It has been refused once probably because 2FA wasn't enabled - but the second attempt has eventually been accepted the next day (2024-11-14), after enabling 2FA. From the moment I sent the first application, I couldn't access Copilot anymore. 

![image](https://github.com/user-attachments/assets/1284ad20-8950-4c96-803f-a787a07caf57)

However, it's been a week since I did that and I still can't access Copilot on VSCode since I tried to renew my subscription. [This page](https://github.com/features/copilot/plans?cft=copilot_li.features_copilot) makes me land on the pricing page instead of telling me that my Copilot access is granted. 

This is displayed on my account; I don't really understand what's going on with the red banner:

![image](https://github.com/user-attachments/assets/0b438d93-0029-4539-aa3e-e44a84dc69fe)

I'd like to have my Copilot access back with the Student Pack, could you help me please?

Best,
@ameu00

**Comment by davevad93** [comment]
Hi @ameu00 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ameu00 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
